library/std/src/sys/pal/unix/time.rs

```diff
@@ -261,8 +261,8 @@ impl Instant {
         // https://www.manpagez.com/man/3/clock_gettime/
         //
         // CLOCK_UPTIME_RAW   clock that increments monotonically, in the same man-
-        //                    ner as CLOCK_MONOTONIC_RAW, but that does not incre-
-        //                    ment while the system is asleep.  The returned value
```

Comment:
ugh

Reply:
Not checking 4 letter words would avoid needing this change.

Reply:
ok.